test/controllers/register/trustees/company/CheckYourAnswersControllerSpec.scala

```diff
@@ -39,7 +39,7 @@ class CheckYourAnswersControllerSpec extends ControllerSpecBase {
   lazy val companyDetailsSection = AnswerSection(
     Some("messages__checkYourAnswers__section__company_details"),
     Seq(
-      AnswerRow("messages__common__cya__name", Seq(companyName), false, companyDetailsRoute),
+      AnswerRow("messages__common__cya__name", Seq("test company name"), false, companyDetailsRoute),
```

Comment:
should be 'test scheme name' not 'test company name'

Reply:
not required :)